src/main/java/com/cs56fitnessapp/models/StrengthTraining.java

```diff
@@ -0,0 +1,54 @@
+package com.cs56fitnessapp.models;
+
+import  com.cs56fitnessapp.models.workout.Workout;
+import java.util.ArrayList;
+import java.time.LocalDate;
+
+/**
+ *  @author Daniel Cervantes
+ *  @author Ksenia Koldaeva
+ *
+ * Date created: 10/18/17
+ * Last updated: 10/18/17
+ */
+
+
+public class StrengthTraining extends Workout {
+
+
+    private ArrayList<String> exerciseList = new ArrayList<>(){{
```

Comment:
Per UML `exerciseList` is of type `ArrayList< StrengthExercise >` not `ArrayList<String>`. Please correct.